go.mod

```diff
@@ -6,6 +6,8 @@ require (
 	github.com/btcsuite/btcd v0.20.1-beta
 	github.com/coreos/go-semver v0.3.0
 	github.com/dgraph-io/badger/v2 v2.0.3
+	github.com/ethereum/go-ethereum v1.9.12 // indirect
```

Comment:
this definitely should not be here. i guess it is used by `bmt` repo, so a cleanup might be needed